style.css

```diff
@@ -0,0 +1,124 @@
+head {
+  box-sizing: border-box;
+}
+
+body {
+  font-family: 'Poppins', sans-serif;
+  font-size: 1rem;
+  font-weight: 400;
+  line-height: 1.4;
+  color: white;
+  margin: 0;
+}
+
+body::before {
+  content: '';
+  position: fixed;
+  top: 0;
+  left: 0;
+  height: 100%;
+  width: 100%;
+  z-index: -1;
+  background: var(--color-darkblue);
+  background-image: linear-gradient(115deg, rgba(58, 58, 158, 0.8), rgba(136, 136, 206, 0.7));
+  background-size: cover;
+  background-repeat: no-repeat;
+  background-position: center;
+}
+
+.container {
+  width: 100%;
+  max-width: 720px;
+  margin: 3.125rem auto 0 auto;
+}
+
+.header {
+  padding: 0 0.625rem;
+  margin-bottom: 1.875rem;
+}
```

Comment:
- [ ] In your suggestion, you didn't give reasons why the author should make the text aligned. Kindly give reasons why the author should do so. For Example, **_"you should make the text aligned because it is a core requirement of the project"._**
- [ ] Your suggestions appear like an order, kindly rephrase your suggestion and make it sound polite. You could use kind words like _**'kindy'**_, _**please**_. _**I think...**_ and other kind words.

- This is a polite suggestion: "I think it would be better to use semantic tags" or "Kindly use semantic tags"

- This reads as an order: "Use semantic tags"

#### When making the change to your inline comment, kindly edit the old one. Please don't create a new comment for it. 👍